jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/OpenshiftHelperStatusTest.java

```diff
@@ -0,0 +1,121 @@
+                    {"IsErrorTrue", "Error", true},
+                    {"IsCancelledTrue", "Cancelled", true},
+                    {"IsNotCompleteFalse", "not Complete", false}
+            });
+        }
+
+        @Parameterized.Parameter
+        public String description;
+        @Parameterized.Parameter(1)
+        public String input;
+        @Parameterized.Parameter(2)
+        public Boolean expected;
+
+        @Test
+        public void testIsFinishedUsingParametrizedTest() {
+            assertThat(OpenshiftHelper.isFinished(input)).isEqualTo(expected);
+        }
+    }
+    @RunWith(Parameterized.class)
+    public static class Cancelled {
+        @Parameterized.Parameters(name = "{0}: isCancelled({1})={2}")
+        public static Collection<Object[]> data() {
+            return Arrays.asList(new Object[][]{
+                    {"IsCancelledTrue", "Cancelled", true},
+                    {"IsNotCancelledFalse", "not Cancelled", false}
+            });
+        }
+
+        @Parameterized.Parameter
+        public String description;
+        @Parameterized.Parameter(1)
+        public String input;
+        @Parameterized.Parameter(2)
+        public Boolean expected;
+
+        @Test
+        public void testIsCancelledUsingParametrizedTest() {
+            assertThat(OpenshiftHelper.isCancelled(input)).isEqualTo(expected);
+        }
+    }
+    @RunWith(Parameterized.class)
+    public static class Failed {
+        @Parameterized.Parameters(name = "{0}: isFailed({1})={2}")
+        public static Collection<Object[]> data() {
+            return Arrays.asList(new Object[][]{
+                    {"IsFailTrue", "Fail", true},
+                    {"IsErrorTrue", "Error", true},
+                    {"IsNullFalse","null",false}
+            });
+        }
+
+        @Parameterized.Parameter
+        public String description;
+        @Parameterized.Parameter(1)
+        public String input;
+        @Parameterized.Parameter(2)
+        public Boolean expected;
+
+        @Test
+        public void testIsFailedUsingParametrizedTest() {
+            assertThat(OpenshiftHelper.isFailed(input)).isEqualTo(expected);
+        }
+    }
+    @RunWith(Parameterized.class)
+    public static class Completed {
+        @Parameterized.Parameters(name = "{0}: isCompleted({1})={2}")
+        public static Collection<Object[]> data() {
+            return Arrays.asList(new Object[][]{
+                    {"IsCompleteTue", "Complete", true},
+                    {"IsNotCompleteFalse", "not Complete", false}
+            });
+        }
+
+        @Parameterized.Parameter
+        public String description;
+        @Parameterized.Parameter(1)
+        public String input;
+        @Parameterized.Parameter(2)
+        public Boolean expected;
+
+        @Test
+        public void testIsCompletedUsingParametrizedTest() {
+            assertThat(OpenshiftHelper.isCompleted(input)).isEqualTo(expected);
```

Comment:
Please avoid calculating values in `assertThat`. Consider extracting a new variable instead.

Reply:
sure @adriannowak , changed!